tardis/plasma/properties/transition_probabilities.py

```diff
@@ -330,11 +330,20 @@ def calculate(
             names=p_deactivation.index.names,
         )
         fb_cooling_probs.insert(0, "lines_idx", continuum_idx)
-        fb_cooling_probs.insert(
-            0,
-            "transition_type",
-            level_idxs2transition_idx.at[("k", "bf"), "transition_type"],
-        )
+        # Since 'at' seems to be faster than 'loc' we try it first
+        # However, if the index is not unique, 'at' will raise a ValueError
+        try:
+            fb_cooling_probs.insert(
+                0,
+                "transition_type",
```

Comment:
but these are different - right? one inserts `value[0]` the other inserts the value. Why are there sometimes multiple? are they duplicate? Should we drop them?

Reply:
This bit is just a workaround to avoid an error from this. This should (probably) be removed before merging. I suspect that this is only a symptom of the issue. The duplicates are already present in ``atomic.py``.